pkg/setting/setting_unified_alerting.go

```diff
@@ -104,6 +104,10 @@ type UnifiedAlertingStateHistorySettings struct {
 	Enabled       bool
 	Backend       string
 	LokiRemoteURL string
+	// LokiTenantID and LokiPassword are used for basic auth if both
```

Comment:
After reading https://grafana.com/docs/loki/latest/operations/authentication/ I wonder if we should just name it BasicAuthUser/BasicAuthPassword and add a third field for TenantID. Although in grafana cloud we do map tenant/apiKey to the basic auth credentials, other users (on-prem) aren't guaranteed to be running it that way.

Reply:
True, good catch. I'll change it.